basic markup and styles

StasioChaplin opened the pull request:
базовая разметка, базовые стили.
разметки внутри карточек пока нет, хочу дождаться картинок. а live server пока вообще выдает голую страницу

StasioChaplin committed:
basic markup and styles

базовая разметка, базовые стили.
разметки внутри карточек пока нет, хочу дождаться картинок. а live server пока вообще выдает голую страницу

StasioChaplin committed:
2nd commit

добавил некоторые характеристики текста.

StasioChaplin committed:
Merge branch 'main' of https://github.com/IgorKonoval/goit-teamwork-14 into origin/vegetables

StasioChaplin committed:
finalizing

IgorKonoval commented on src/partials/vegetables.html:
```diff
@@ -0,0 +1,393 @@
+<section class="vegetables">
+  <div class="container">
+    <h2 class="veg-title">oRGANIC vegetables</h2>
```
Пропиши текст нормально

IgorKonoval commented on src/partials/vegetables.html:
```diff
@@ -0,0 +1,393 @@
+              <img src="../img/vegetables/carrot.png" alt="Carrot" />
+            </picture>
+          </div>
+          <p class="veg-price">100 UAH / kg</p>
```
текст прописывается согласно орфографии

IgorKonoval commented on src/partials/vegetables.html:
```diff
@@ -0,0 +1,393 @@
+              />
+              <source
+                srcset="
+                  ../img/vegetables/brussel-mb.png    1x,
```
на всех ссылках к рисункам поставить ОДНУ точку

IgorKonoval reviewed (changes requested):
во всех ссылках на картинки ставить ОДНУ точку
в некоторых местах текст большими буквами, так неправильно
из-за жестко заданной высоты у тебя некрасиво растягивается секция даже при 320 уде не очень

StasioChaplin committed:
review correction

Убрал кривые буквы
Убрал лишние точки в ссылках на картинки
Задал медиа до 374пикселей макс-хайт карточке, чтоб тянулось, а с 375 фиксированную высоту, чтоб по макету

IgorKonoval commented on src/css/vegetables.css:
```diff
@@ -80,6 +79,18 @@
 
 /* ==================== MEDIA ==================== */
 
+@media screen and (max-height: 374px) {
```
видали це правило

IgorKonoval reviewed (changes requested):
Прибери це правило, воно не працює. Шо з ним шо без нього однаково

StasioChaplin committed:
last corrections

IgorKonoval committed:
Merge branch 'main' into origin/vegetables
